modernizer-annotation-processor/src/test/java/org/gaul/modernizer_annotation_processor/ModernizerAnnotationProcessorTest.java

```diff
@@ -0,0 +1,361 @@
+/*
+ * Copyright 2014-2018 Andrew Gaul <andrew@gaul.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gaul.modernizer_annotation_processor;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public final class ModernizerAnnotationProcessorTest {
+
+    private List<String> ignoreClasses = new ArrayList<String>();
+    private List<String> ignoreMethods = new ArrayList<String>();
```

Comment:
Instead of simply maintaining a set here, we could build a helper class that's used by both the test and the actual plugin. We could initialize it with this file. Then we could call `ignoreMethodHelper.shouldIgnore(input)` and get a test that tests end to end behavior, rather than checking the internal representation.

Reply:
I guess this is already done in your modernizer test. Maybe we could have read the file instead of explicitly adding things to ignore.